man/systemd.mount.xml

```diff
@@ -366,9 +366,9 @@
       <varlistentry>
         <term><varname>Options=</varname></term>
 
-        <listitem><para>Mount options to use when mounting. This takes
-        a comma-separated list of options. This setting is
-        optional.</para></listitem>
+        <listitem><para>Mount options to use when mounting. This takes a comma-separated list of options. This setting
+        is optional. Note that the usual specifier expansion is applied to this setting, literal percent characters
+        should hence be written as <literal>%%</literal>.</para></listitem>
```

Comment:
https://github.com/systemd/systemd/pull/4835/commits/d107589cd2767c95fb1c794d4f90832301a297f5#diff-815691af24aeb3dba59fb6414f2ec484L357

Shouldn't we escape (and document) `What` too?